Header header
string group_name
moveit_msgs/RobotState query_configuration
float32 radius

